pyclient/pydeephaven/table.py

```diff
@@ -0,0 +1,682 @@
+#
+#  Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+
+from __future__ import annotations
+
+from typing import List
+
+import pyarrow
+
+from pydeephaven._combo_aggs import ComboAggregation
+from pydeephaven.constants import MatchRule, AggType, SortDirection
+from pydeephaven.dherror import DHError
+
+
+class Table:
+    """ A Table object represents a reference to a in-memory table on the server. It is the core data structure of
+    Deephaven and supports a rich set of operations such as filtering, sorting, aggregating, joining, snapshotting etc.
+
+    Note, an application should never instantiate a Table object directly. Table objects are always provided through
+    factory methods such as Session.empty_table(), or import/export methods such as Session.import_table(),
+    open_table(), or any of the Table operations.
+
+    Attributes:
+        is_closed (bool): check if the table has been closed on the server
+    """
+
+    def __init__(self, session=None, ticket=None, schema_header=b'', size=0, is_static=True, schema=None):
```

Comment:
Why have default values for size, is_static instead of None?

Reply:
Would it be better to have a constructor that takes a "Response"?

Reply:
I agree None is a better default value for both size and is_static.
grpc table API 'Response' isn't available in the case of doPut